ale_linters/ansible/ansible_lint.vim

```diff
@@ -43,7 +56,7 @@ endfunction
 
 call ale#linter#Define('ansible', {
 \   'name': 'ansible',
-\   'executable': 'ansible',
-\   'command': 'ansible-lint -p %t',
+\   'executable_callback': 'ale_linters#ansible#ansible_lint#GetExecutable',
+\   'command_callback': 'ale_linters#ansible#ansible_lint#GetCommand',
```

Comment:
Write `'command': '%e -p %t',` instead. which is equivalent.

Reply:
Done